rice-namenode/source/ClientNamenodeProtocolImpl.cc

```diff
@@ -83,7 +90,9 @@ std::string ClientNamenodeTranslator::create(std::string input) {
 	CreateResponseProto res;
 	if (zk.create_file(req, res))
 		lease_manager.addLease(req.clientname(), req.src());
-	LOG(INFO) << CLASS_NAME <<  res.DebugString();
+	else {
+		throw GetErrorRPCHeader("Could not create file", "IOException");
```

Comment:
Does not make sense to have IOException here, bcause thats not what it is. Should just leave it blank so we don't confuse.